pyserini/index/lucene/_base.py

```diff
@@ -535,3 +537,36 @@ def stats(self) -> Dict[str, int]:
             index_stats_dict[term] = index_stats_map.get(term)
 
         return index_stats_dict
+
+    def dump_documents_BM25(self, file_path="collections/documents_bm25_dump.json"):
```

Comment:
Few comments here:
- I don't think output file should have a default value; i.e., the caller should explicitly pass something in.
- please expose the b1 and k values from the underlying method, but keep the standard defaults of 0.9 and 0.4